packages/neon/neon/lib/src/router.dart

```diff
@@ -98,6 +98,14 @@ class LoginRoute extends GoRouteData {
   Widget build(final BuildContext context, final GoRouterState state) => LoginPage(serverURL: server);
 }
 
+@immutable
+class AddAccountRoute extends GoRouteData {
+  const AddAccountRoute();
+
+  @override
+  Widget build(final BuildContext context, final GoRouterState state) => const LoginPage();
+}
```

Comment:
Why is this separate route needed? I see it hides the serverURL paramter, but as far as I can tell it doesn't have any other effect?

Reply:
So we can have a backbutton and discard the login.

I guess that the backbutton calls Navigstor.of(context).pop() internally and /login is a top level route so we can't pop from it.

Reply:
Ah that makes sense. Thanks for the explanation!